<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>517</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: None;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="edit_parameters_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>330</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #495867;
border: 3px solid;
border-color: #EAEAEA;
padding: 2px;
font:18pt &quot;Arial&quot;;
color: #EAEAEA;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>MODIFY 
VALUES</string>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>32</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;
color: #EAEAEA;
border: none;</string>
    </property>
    <property name="text">
     <string>SET</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;
color: #EAEAEA;
border: none;</string>
    </property>
    <property name="text">
     <string>MEASURED</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="message_log_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #EAEAEA;
font: 12pt &quot;Arial&quot;;
padding: 8px;
color: #292F36;
border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>Relevant Messages can be displayed here</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">background-color: #EAEAEA;
font: 16pt &quot;Nimbus sans L&quot;;
color: #292F36;
border: 3px solid;
border-color: #495867;
border-radius: 4px;</string>
       </property>
       <property name="text">
        <string>TV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">background-color: #EAEAEA;
font: 16pt &quot;Nimbus sans L&quot;;;
color: #292F36;
border: 3px solid;
border-color: #495867;
border-radius: 4px;
</string>
       </property>
       <property name="text">
        <string>BPM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">background-color: #EAEAEA;
font: 16pt &quot;Nimbus sans L&quot;;
color: #292F36;
border: 3px solid;
border-color: #495867;
border-radius: 4px;
</string>
       </property>
       <property name="text">
        <string>I:E</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">background-color: #495867;
border: 3px solid;
border-color: #EAEAEA;
font: 16pt &quot;Nimbus sans L&quot;;
color: #EAEAEA;
border-radius: 4px;</string>
       </property>
       <property name="text">
        <string>PIP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">background-color: #495867;
border: 3px solid;
border-color: #EAEAEA;
font: 16pt &quot;Nimbus sans L&quot;;
color: #EAEAEA;
border-radius: 4px;</string>
       </property>
       <property name="text">
        <string>PEEP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="styleSheet">
        <string notr="true">background-color: #495867;
border: 3px solid;
border-color: #EAEAEA;
font: 16pt &quot;Nimbus sans L&quot;;
color: #EAEAEA;
border-radius: 4px;
</string>
       </property>
       <property name="text">
        <string>PLAT</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>61</width>
      <height>410</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="set_TV_label">
       <property name="styleSheet">
        <string notr="true">color: #292F36;
background-color: None;
border-style: none;
font: 32pt &quot;Nimbus sans L&quot;;</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="set_BPM_label">
       <property name="styleSheet">
        <string notr="true">color: #292F36;
background-color: None;
border-style: none;
font: 32pt &quot;Nimbus sans L&quot;;</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="set_IE_label">
       <property name="styleSheet">
        <string notr="true">color: #292F36;
background-color: None;
border-style: none;
font: 32pt &quot;Nimbus sans L&quot;;</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="set_PIP_label">
       <property name="styleSheet">
        <string notr="true">color: #EAEAEA;
background-color: None;
border-style: none;
font: 32pt &quot;Nimbus sans L&quot;;</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="set_PEEP_label">
       <property name="styleSheet">
        <string notr="true">color: #EAEAEA;
background-color: None;
border-style: none;
font: 32pt &quot;Nimbus sans L&quot;;</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="set_PLAT_label">
       <property name="styleSheet">
        <string notr="true">color: #EAEAEA;
background-color: None;
border-style: none;
font: 32pt &quot;Nimbus sans L&quot;;</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>61</width>
      <height>410</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="measure_TV_label">
       <property name="styleSheet">
        <string notr="true">color: #292F36;
background-color: None;
border-style: none;
font: 32pt &quot;Nimbus sans L&quot;;</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="measure_BPM_label">
       <property name="styleSheet">
        <string notr="true">color: #292F36;
background-color: None;
border-style: none;
font: 32pt &quot;Nimbus sans L&quot;;</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="measure_IE_label">
       <property name="styleSheet">
        <string notr="true">color: #292F36;
background-color: None;
border-style: none;
font: 32pt &quot;Nimbus sans L&quot;;</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="measure_PIP_label">
       <property name="styleSheet">
        <string notr="true">color: #EAEAEA;
background-color: None;
border-style: none;
font: 32pt &quot;Nimbus sans L&quot;;</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="measure_PEEP_label">
       <property name="styleSheet">
        <string notr="true">color: #EAEAEA;
background-color: None;
border-style: none;
font: 32pt &quot;Nimbus sans L&quot;;</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="measure_PLAT_label">
       <property name="styleSheet">
        <string notr="true">color: #EAEAEA;
background-color: None;
border-style: none;
font: 32pt &quot;Nimbus sans L&quot;;</string>
       </property>
       <property name="text">
        <string>00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>551</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="PlotWidget" name="pressure_graph" native="true">
       <widget class="PlotWidget" name="volume_graph" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-5</y>
          <width>549</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-40</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #292F36</string>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>edit_parameters_button</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>message_log_label</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>layoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
